cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -187,6 +188,14 @@ func isListResource(msg *beehiveModel.Message) bool {
 	return false
 }
 
+func isVolumeMessage(msg *beehiveModel.Message) bool {
+	op := msg.GetOperation()
+	return op == commonconst.CSIOperationTypeCreateVolume ||
+		op == commonconst.CSIOperationTypeDeleteVolume ||
+		op == commonconst.CSIOperationTypeControllerPublishVolume ||
+		op == commonconst.CSIOperationTypeControllerUnpublishVolume
+}
```

Comment:
We can just call this func in L160-164, the will send this message to Listqueue, which will be sent successfully.  Ref: https://github.com/kubeedge/kubeedge/pull/3457/files#diff-7c1e619cc873b4c19c88bb612e6a476e9fa1af8cf21d19b43a3c374d0b201cbfR160-R163